ui/guest/src/components/GuestProxy.js

```diff
@@ -191,7 +195,7 @@ export function GuestProxy(props) {
           // Update attribute(s)
           // $el.attr('data-craftercms-index', newIndex);
```

Comment:
What's up with these bits of commented code? (lines 129, 196, 201)

Reply:
there are your commets